packages/pwa-kit-dev/src/configs/webpack/config.js

```diff
@@ -340,6 +340,8 @@ const ruleForBabelLoader = (babelPlugins) => {
         id: 'babel-loader',
         test: /(\.js(x?)|\.ts(x?))$/,
         // NOTE: Because our extensions are just folders containing source code, we need to ensure that the babel-loader processes them.
+        // By default babel doesn't process files in "node_modules" folder, so here we will ensure they are included.
+        exclude: /node_modules\/(?!(@?[^/]+\/)?extension-)[^/]+\/.*$/i,
```

Comment:
Dumb question, we have `exclude` here and `ignore` in the babel config? How are they working with each other? I recall they can't be used on a same directory like `node_modules` because exclude is exclusive to the folder and will override `ignore`.

Reply:
These are different configurations that don't work with each other. One is the Webpack configuration for babel-loader, and the other is the Babel configuration for babel-node, where we cannot use the exclude option. They are used in separate stages.